sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+```
+
+Every nf-core pipeline has a full list of parameters on the nf-core website. You will be shown a description and the type of the parameter when viewing these parameters. Some parameters will also have additional text to help you understand how a parameter should be used. See the [parameters page of the nf-core rnaseq pipeline](https://nf-co.re/rnaseq/3.14.0/parameters/).
+
+### Default configuration files
+
+All parameters have a default configuration that is defined using the `nextflow.config` file in the root of the pipeline directory. Many parameters are set to `null` or `false` by default and are only activated by a profile or config file.
+
+There are also several `includeConfig` statements in the `nextflow.config` file that are used to include additional config files from the `conf/` folder. Each additional `.config` file contains categorized configuration information for your pipeline execution, some of which can be optionally included as profiles:
+
+- `base.config`
+  - Included by the pipeline by default
+  - Generous resource allocations using labels
+  - Does not specify any method for software dependencies and expects software to be available (or specified elsewhere)
+- `igenomes.config`
+  - Included by the pipeline by default
+  - Default configuration to access reference files stored on AWS iGenomes
+- `modules.config`
+  - Included by the pipeline by default
+  - Module-specific configuration options (both mandatory and optional)
+- `test.config`
+  - Only included if specified as a profile
+  - A configuration profile to test the pipeline with a small test dataset
+- `test_full.config`
+  - Only included if specified as a profile
+  - A configuration profile to test the pipeline with a full-size test dataset
+
+:::note
+Some configuration files contain the definition of profiles that can be flexibly applied. For example, the `docker`, `singularity`, and `conda` profiles are defined in the `nextflow.config` file in the pipeline project directory.
+:::
+
+Profiles are sets of configuration options that can be flexibly applied to a pipeline.
+They are also commonly defined in the `nextflow.config` file in the root of the pipeline directory.
+
+Profiles used by nf-core pipelines can be broadly categorized into two groups:
+
+- Software management profiles
+  - Profiles for the management of software dependencies using container or environment management tools, for example, `docker`, `singularity`, and `conda`.
+- Test profiles
+  - Profiles to execute the pipeline with a standardized set of test data and parameters, for example, `test` and `test_full`.
+
+nf-core pipelines are required to define software containers and environments that can be activated using profiles. Although it is possible to run the pipelines with software installed by other methods (e.g., environment modules or manual installation), using container technology is more sharable, convenient, and reproducible.
+
+### Shared configuration files
```

Comment:
Iwonder if this should berenamed to 'shared infrastructure files' or something similar, as that is waht they are used for, not som uch sharing e.g. pipeline parameters between users

Reply:
I think the focus here is the source of configuration rather than the bias towards these being used for infrastruture.